cli/login.go

```diff
@@ -32,7 +37,7 @@ func (p *LoginCmd) SetFlags(f *flag.FlagSet) {
 
 func (p *LoginCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
 	reader := bufio.NewReader(os.Stdin)
-	tokenUrl, err := ybconfig.ManagementUrl("user/apitoken")
+	tokenUrl, err := ybconfig.ManagementUrl(APP_SETTINGS)
```

Comment:
Maybe just rename this method to ybconfig.UserSettingsUrl()? Clearer and keeps the related bits closer together